odkx-src/build-app.rst

```diff
@@ -21,7 +21,7 @@ This will walk you through the steps of building a Data Management Application f
 ODK-X Data Management Applications
 ---------------------------------------
 
-The ODK-X Android tools (ODK-X Survey, ODK-X Tables, ODK-X Services, ODK-X Scan, ODK-X Sensors Framework, and various ODK-X Sensor implementations) are Android Application Packages (APKs) that are designed to work together to create a coherent tailored application experience for an end-user.
+The ODK-X Android tools (:doc:`survey-using`, :doc:`tables`, :doc:`services`, :doc:`scan-intro`, `ODK-X Sensors Framework <https://github.com/odk-x/sensorsframework>`_, and various ODK-X Sensor implementations) are Android Application Packages (APKs) that are designed to work together to create a coherent tailored application experience for an end-user.
```

Comment:
I wonder if we should cut the mention of scan and sensors from here since they are not actively supported... @wbrunette thoughts?

Reply:
Discussed in call...

Reply:
make e.g.

Reply:
make for example

Reply:
@EstherPhebe thanks for deleting the additional tools from here, not sure you got @wbrunette "For example" added though.

Reply:
Agree that we should add for example...